pkg/tests/ossm/smm_test.go

```diff
@@ -83,6 +84,32 @@ func TestSMMRAutoCreationAndDeletion(t *testing.T) {
 	})
 }
 
+func TestSMMReconciliation(t *testing.T) {
```

Comment:
This doesn't need to run for all SMCP versions, as it doesn't depend on the SMCP version. How do I make this test run only once? Are there any other tests like this?

Reply:
Check if SMCP versions is the newest?